### DISCUSSION TITLE
👋 Welcome to our GitHub Discussion!

### DISCUSSION THREAD

**Discussion by zheeeng** [galacean/effects-runtime#3] [Announcements]
## 👋 欢迎来到我们的 GitHub Discussion！

这是一个讨论和交流的空间，无论您是新手还是经验丰富的开发者，都可以在这里找到解答或与其他人交流。

我们非常欢迎您在这里：
* 提出对项目的改进建议或功能请求；
* 分享您的想法、知识与学习资源；
* 与 Galacean Effects 的开发成员、大众设计师、动效开发者互动。

如果遇到了 Galacean Effects Runtime 相关的问题，欢迎移步 Issue 区，将会有相关的人员进行解答。

请在发帖前注意一些事项：

* 尽量使用简洁的标题，以便其他人快速了解您的问题或主题。
* 尊重他人的意见和观点，避免使用侮辱性、攻击性或令人不悦的语言。如果有争议，请友好地讨论。
* 避免在讨论中散布垃圾信息、政治相关内容、招聘消息、广告或其他无关内容。

感谢您的参与和贡献！我们期待与您一起建立一个开放、友好和有益的讨论社区。

---

## 👋 Welcome to our GitHub Discussion!
This is a space for discussion and exchange, where both newcomers and experienced developers can find answers or interact with others.

We are very glad to have you here to:

* Suggest improvements or feature requests for the project;
* Share your ideas, knowledge, and learning resources;
* Interact with Galacean Effects developers, general designers, and motion effects developers. 

If you encounter any issues related to Galacean Effects Runtime, feel free to head over to the Issues section, where relevant personnel will assist you.

Please keep in mind some points before posting:

* Try to use concise titles so that others can quickly understand your question or topic.
* Respect others' opinions and viewpoints, avoid using insulting, aggressive, or displeasing language. If there are disagreements, please discuss politely.
* Avoid spreading spam, political content, job listings, advertisements, or other irrelevant content in the discussions.

Thank you for your participation and contribution! We look forward to building an open, friendly, and beneficial discussion community with you.

**Comment by lizyChy0329** [comment]
发布【内测中】
更新于 2024-04-09 15:28

内测中是什么意思？意味着现在不能使用或者不稳定？

而且官方跳转页面时，不会自动 goTop ，非常难受

**Reply by yiiqii** [reply]
发布在内测中，非内测用户直接使用导出 zip 格式即可，不影响使用

**Reply by yiiqii** [reply]
而且官方跳转页面时，不会自动 goTop ，非常难受
————
收到，我们优化一下

**Comment by lizyChy0329** [comment]
灵感中心 - 618信用卡 
打开提示 “模板数据格式有误”

灵感中心 - 获得红包
打开项目后，画板区域没有任何东西显示，其他区域正常

打开项目 - 点击预览
为什么不能预览当前项目，而是提示“拖拽上传离线产物XXXXX”

**Reply by yiiqii** [reply]
1、灵感资源有问题，已更新
2、这边都正常，再试试看看？
3、实时预览当前也在内测中，所以非内测用户暂时只能使用离线预览：导出 zip 产物，拖入预览区域即可

感谢反馈🙏

**Reply by lizyChy0329** [reply]
你那个 tiny.js 为啥 github 搜不到了，感觉非常易用

**Reply by lizyChy0329** [reply]
Browser: Chrome 128
System: Windows 10

![image](https://github.com/user-attachments/assets/8bb30da6-e835-4d4b-9a8b-f652acde634b)

**Reply by yiiqii** [reply]
> 你那个 tiny.js 为啥 github 搜不到了，感觉非常易用

https://tinyjs.net/

**Reply by lizyChy0329** [reply]
> > 你那个 tiny.js 为啥 github 搜不到了，感觉非常易用
> 
> [tinyjs.net](https://tinyjs.net/)

我看21年就没维护了，有源代码吗？

**Reply by yiiqii** [reply]
> > > 你那个 tiny.js 为啥 github 搜不到了，感觉非常易用
> > 
> > 
> > [tinyjs.net](https://tinyjs.net/)
> 
> 我看21年就没维护了，有源代码吗？

代码未开源

**Reply by yiiqii** [reply]
> Browser: Chrome 128 System: Windows 10
> 
> ![image](https://private-user-images.githubusercontent.com/103976180/369327596-8bb30da6-e835-4d4b-9a8b-f652acde634b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4MjEzNDEsIm5iZiI6MTcyNjgyMTA0MSwicGF0aCI6Ii8xMDM5NzYxODAvMzY5MzI3NTk2LThiYjMwZGE2LWU4MzUtNGQ0Yi05YThiLWY2NTJhY2RlNjM0Yi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMFQwODMwNDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZmMzZTgxYjExNGM4MjIwMDg0ZjkyMmQ0NDBlMzIzN2EwN2JjODk5NzcxZjczNjQ4M2Y1N2M1NmQwZDg4M2QzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.2KbGTaWU3ha_I_Aa6sbRdR04S0NZ0HFutCnpUv4PY1g)

点播放没有任何东西？

**Reply by lizyChy0329** [reply]
:) 原来要点播放...我以为坏了，不应该停留在最后一帧么

**Reply by yiiqii** [reply]
> :) 原来要点播放...我以为坏了，不应该停留在最后一帧么

Galacean Effects 默认停在第一帧
